fcrepo-fixity/src/test/resources/OSGI-INF/blueprint/blueprint-integration-test.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is the rationale for having both `blueprint-integration-test.xml` and `blueprint-test.xml` since they are identical?